samples/gpu/farneback_optical_flow.cpp

```diff
@@ -59,8 +59,8 @@ int main(int argc, char **argv)
 
     string pathL = cmd.get<string>("left");
     string pathR = cmd.get<string>("right");
-    if (pathL.empty()) cout << "Specify left image path\n";
-    if (pathR.empty()) cout << "Specify right image path\n";
+    if (pathL.empty()) pathL = "../data/basketball1.png";
```

Comment:
Can these paths be added as default parameter values (like you did in `pyrlk_optical_flow.cpp`) or it will cause a problem?